packages/cdk/lib/roles/policies/nextflow-submit-job-batch-policy.ts

```diff
@@ -0,0 +1,24 @@
+import * as iam from "monocdk/aws-iam";
+
+export interface NextflowSubmitJobBatchPolicyProps {
+  account: string;
+  region: string;
+  submitJobPolicyArns: string[];
+}
+
+export class NextflowSubmitJobBatchPolicy extends iam.PolicyDocument {
+  constructor(props: NextflowSubmitJobBatchPolicyProps) {
+    const nextflowJobArn = `arn:aws:batch:${props.region}:${props.account}:job-definition/nf-ubuntu:*`;
+
+    super({
+      assignSids: true,
+      statements: [
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          actions: ["batch:SubmitJob"],
```

Comment:
Nextflow will Cancel/Terminate jobs if the parent job is cancelled too. We'll probably want `CancelJob/TerminateJob` permissions.

Reply:
Will test the workflow stop command and make sure everything works 👍